modules/mongodb/templates/mongodb-backup-s3.erb

```diff
@@ -17,65 +17,109 @@ date_format = '%Y-%m-%d:%H:%M:%S'
 mongo_server = '<%= @server %>'
 key_fingerprint = '<%= @private_gpg_key_fingerprint %>'
 s3_bucket = '<%= @s3_bucket %>'
-standalone = '<%= @standalone %>'
+<% if standalone -%>
```

Comment:
I really dislike this approach. I don't like the idea that the application code changes based upon puppet values. I'd much rather see this done as command line arguments to the script and have the invoking cronjob pass in the arguments that come from the hiera lookups.